chapters/file-information.md

```diff
@@ -795,7 +795,8 @@ EXAMPLE 2 RDF: Property `spdx:fileDependency` in class `spdx:File`
 [pip-vcs]: https://pip.pypa.io/en/latest/reference/pip_install.html#vcs-support
 [Red Hat]: https://www.redhat.com/
 [rfc3986]: https://tools.ietf.org/html/rfc3986
-[SHA-1]: https://tools.ietf.org/html/rfc3174
+[SHA1]: https://tools.ietf.org/html/rfc3174
+[SHA1-DC]: https://github.com/cr-marcstevens/sha1collisiondetection
```

Comment:
Is the intent that the SHA1-DC is a SHA1 that the SPDX creator states has passed the collision detection?  If so, I'm not sure this is really a different HASH algorithm from SHA1.